task/github-add-comment/0.4/tests/fixtures/github-post-comment.yaml

```diff
@@ -13,4 +13,4 @@ headers:
ps://api.github.com/repos/tektoncd/plumbing/issues/111",      "id": 111111111111,      "node_id": "abcd==",      "user": {        "login": "johndoe",        "id": 12345,        "node_id": "MDQ6VXNlcjk4OTgw",        "avatar_url": "https://avatars.githubusercontent.com/u/12345?v=4",        "gravatar_id": "",        "url": "https://api.github.com/users/johndoe",        "html_url": "https://github.com/johndoe",        "followers_url": "https://api.github.com/users/johndoe/followers",        "following_url": "https://api.github.com/users/johndoe/following{/other_user}",        "gists_url": "https://api.github.com/users/johndoe/gists{/gist_id}",        "starred_url": "https://api.github.com/users/johndoe/starred{/owner}{/repo}",        "subscriptions_url": "https://api.github.com/users/johndoe/subscriptions",        "organizations_url": "https://api.github.com/users/johndoe/orgs",        "repos_url": "https://api.github.com/users/johndoe/repos",        "events_url": "https://api.github.com/users/johndoe/events{/privacy}",        "received_events_url": "https://api.github.com/users/johndoe/received_events",        "type": "User",        "site_admin": false      },      "created_at": "2021-01-18T12:56:24Z",      "updated_at": "2021-01-18T12:56:24Z",      "author_association": "MEMBER",      "body": "/kind feature",      "performed_via_github_app": null    },    {      "url": "https://api.github.com/repos/tektoncd/plumbing/issues/comments/2222222222",      "html_url": "https://github.com/tektoncd/plumbing/pull/111#issuecomment-2222222222",      "issue_url": "https://api.github.com/repos/tektoncd/plumbing/issues/111",      "id": 2222222222,      "node_id": "MDEyOklzc3VlQ29tbWVudDc2MjIzNDExNg==",      "user": {        "login": "johndoe",        "id": 12345,        "node_id": "123abc123",        "avatar_url": "https://avatars.githubusercontent.com/u/12345?v=4",        "gravatar_id": "",        "url": "https://api.github.com/users/johndoe",        "html_url": "https://github.com/johndoe",        "followers_url": "https://api.github.com/users/johndoe/followers",        "following_url": "https://api.github.com/users/johndoe/following{/other_user}",        "gists_url": "https://api.github.com/users/johndoe/gists{/gist_id}",        "starred_url": "https://api.github.com/users/johndoe/starred{/owner}{/repo}",        "subscriptions_url": "https://api.github.com/users/johndoe/subscriptions",        "organizations_url": "https://api.github.com/users/johndoe/orgs",        "repos_url": "https://api.github.com/users/johndoe/repos",        "events_url": "https://api.github.com/users/johndoe/events{/privacy}",        "received_events_url": "https://api.github.com/users/johndoe/received_events",        "type": "User",        "site_admin": false      },      "created_at": "2021-01-18T12:57:02Z",      "updated_at": "2021-01-18T12:57:02Z",      "author_association": "MEMBER",      "body": "A comment to replace<!-- TEST_TAG123 -->",      "performed_via_github_app": null    }]'
```

Comment:
Heh, this is what I was trying to avoid 🙏  i.e. embed a large JSON in a one line, since it's not very readable / maintainable.